### ISSUE TITLE
SomeQuestion

### ISSUE THREAD

**Issue by huangtao36** [huangzh13/StyleGAN.pytorch#2]
Thanks for your good work. I train it from sketch with celebA, but I can't get good quality images. I have some questions and hope you can help.
- How long have you trained a model? and How much GPU memory?
- Have you tried StyleGAN2?
- Are there any tricks or is it difficult to train by itself?

**Comment by huangzh13** [comment]
In fact, I also failed on the CelebA dataset, which I think is caused by RelativisticAverageHinge Loss (I originally thought it was a good enough loss function). But please try Logistic Loss, which I added later (loss function in the original paper). According to my experiments, it was successful on the CelebA dataset.

- The training efficiency is really low now, and may need more optimization. It usually takes more than 2 days to train 128 * 128 resolution images.
- I'm trying to implement stylegan2 with a similar structure and make it compatible with official pre-trained models.

**Comment by huangtao36** [comment]
ooh, thanks, this is a great tip, I will try it. And I will try this model on the person images (contain the body), hoping to have a good effect.
